common/script/content/achievements.js

```diff
@@ -0,0 +1,163 @@
+import { each } from 'lodash';
+
+let achievementsData = {
+  dilatoryQuest: {
+    icon: 'achievement-dilatory',
+    titleKey: 'achievementDilatory',
```

Comment:
This would only work for strings that don't take a `{ some: 'thing' }` parameter.